crates/uv/tests/run.rs

```diff
@@ -1368,3 +1368,45 @@ fn run_no_project() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn run_remote_pep723_script() {
+    let context = TestContext::new("3.12").with_filtered_python_names();
+    let mut filters = context.filters();
+    filters.push((
+        r"(?m)^Reading inline script metadata from:.*\.py$",
+        "Reading inline script metadata from: [TEMP_PATH].py",
+    ));
+    uv_snapshot!(filters, context.run().arg("https://gist.githubusercontent.com/manzt/cb24f3066c32983672025b04b9f98d1f/raw/d675b26fe8b3a8e301705af80a53fe5de15a0273/pep723.py"), @r###"
```

Comment:
Just copied the example from [PEP 723](https://peps.python.org/pep-0723/), but realizing now the snapshot will likely change over time (with new versions of deps). Probably better to have a more simple test / and simple dependency.

Reply:
We'll probably create something in the Astral org anyway, thanks!